dali/daliadmin/daadmin.cpp

```diff
@@ -1917,16 +1917,16 @@ void dfsreplication(const char *clusterMask, const char *lfnMask, unsigned redun
 
 void holdlock(const char *logicalFile, const char *mode, IUserDescriptor *userDesc)
 {
-    bool write;
+    AccessMode accessMode;
     if (strieq(mode, "read"))
-        write = false;
+        accessMode = AccessMode::tbdRead;
     else if (strieq(mode, "write"))
-        write = true;
+        accessMode = AccessMode::tbdWrite;
```

Comment:
There's a reference to write below that may be incorrect

Reply:
well spotted.  I'll recheck all the examples/functions.